bedrock/firefox/templates/firefox/channel/ios.html

```diff
@@ -9,8 +9,6 @@
 {% block page_title %}
 {% if l10n_has_tag('firefox_title_meta_bug1345338') %}
   {{ _('Try New Features in a Pre-Release iOS Browser | Firefox') }}
-{% else %}
-  {{_('Test beta versions of Firefox for iOS via Apple’s TestFlight program and help make our mobile browser for iPhone, iPad and iPod touch even better.')}}
```

Comment:
There was no title before, I assume it falls back to a default title? 
https://github.com/mozilla/bedrock/pull/4736/files#diff-a56dc7204957db25760b0dfbf4e0979d

Note that it's not the only case, this page was missing a desc
https://github.com/mozilla/bedrock/pull/4736/files#diff-a56dc7204957db25760b0dfbf4e0979d